xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -34,13 +34,25 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+  <dl>
+    <dt>
+      <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
```

Comment:
Would be good to also escape the XML from the label text using ``$escapetool.xml(...)`` as it should be plain text.

Reply:
Ok got it.